frontend/ux.rst

```diff
@@ -0,0 +1,145 @@
+.. index::
+   single: Symfony UX
+
+The Symfony UX Initiative & Packages
+====================================
+
+Symfony UX is an initiative and set of libraries to seamlessly
+integrate JavaScript tools into your application. For example,
+want to render a chart with `Chart.js`_? Use `UX Chart.js`_ to build the
+chart in PHP. The JavaScript is handled for you automatically.
+
+Behind the scenes, the UX packages leverage `Stimulus`_: a small, but
+powerful library for binding JavaScript functionality to elements on
+your page.
+
+Installing Symfony UX
+---------------------
+
+Before you install any specific UX library, make sure you've installed
+:doc:`Webpack Encore </frontend/encore/installation>`.
+
+If you already have it installed, make sure you have an
+``assets/bootstrap.js`` file (this initializes Stimulus & the UX packages),
+an ``assets/controllers.json`` file (this controls the 3rd party UX packages that
+you've installed) and ``.enableStimulusBridge('./assets/controllers.json')`` in
+your ``webpack.config.js`` file. If these are missing, try upgrading the
+``symfony/webpack-encore-bundle`` Flex recipe. See
+:ref:`Upgrading Flex Recipes <updating-flex-recipes>`.
+
+.. _ux-packages-list:
+
+All Symfony UX Packages
+-----------------------
+
+.. include:: /frontend/_ux-libraries.rst.inc
+
+Stimulus Tools around the World
+-------------------------------
+
+Because Stimulus is used by developers outside of Symfony, many tools
+exist beyond the UX packages:
+
+-   `stimulus-use`_: Add composable behaviors to your Stimulus controllers, like
+    debouncing, detecting outside clicks and many other things.
+
+-   `stimulus-components`_ A large number of pre-made Stimulus controllers, like for
+    Copying to clipboard, Sortable, Popover (similar to tooltips) and much more.
```

Comment:
```suggestion
* `stimulus-use`_: Add composable behaviors to your Stimulus controllers, like
  debouncing, detecting outside clicks and many other things.

* `stimulus-components`_ A large number of pre-made Stimulus controllers, like for
  Copying to clipboard, Sortable, Popover (similar to tooltips) and much more.
```

Not sure if it makes a difference, but let's keep things as simple as possible (least chances of rst-parser not understanding it correctly)